app/_how-tos/aggregate-mcp-tools.md

```diff
@@ -0,0 +1,421 @@
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-mcp-proxy
+      route: mcp-listener
+      config:
+        mode: listener
+        server:
+          tag: mcp-tools
+          timeout: 45000
+        logging:
+          log_statistics: true
+          log_payloads: false
+        max_request_body_size: 32768
+{% endentity_examples %}
+
+## Validate the configuration
+
+### Run the MCP inspector
+
+First, let's run the MCP Inspector to see whether our listener AI MCP Proxy plugin properly aggregates all exposed tools.
+
+1. Execute the following command to run the MCP Inspector:
+
+    ```sh
+    npx @modelcontextprotocol/inspector --mcp-url http://localhost:8000/mcp-listener node path/to/server/index.js
+    ```
+
+1. If successful, you should see the following output in your terminal:
+
+    ```sh
+    Starting MCP inspector...
+    ⚙️ Proxy server listening on localhost:6277
+    🔑 Session token: <YOUR_TOKEN>
+      Use this token to authenticate requests or set DANGEROUSLY_OMIT_AUTH=true to disable auth
+
+    🚀 MCP Inspector is up and running at:
+      http://localhost:6274/?MCP_PROXY_AUTH_TOKEN=<YOUR_TOKEN>
+    ```
+1. The script will automatically open a new browser window with MCP Inspector's UI:
+
+    ![MCP Inspector's UI](/assets/images/ai-gateway/mcp-inspector.png){: style="display:block; margin-left:auto; margin-right:auto; width:70%; border-radius:10px" }
+
+1. Click the **Connect** button on the left.
+
+1. In the **Tools** tile, click the **List tools** button. You should see the following tools available:
+
+    ![MCP tools in MCP Inspector](/assets/images/ai-gateway/mcp-tools.png){: style="display:block; margin-left:auto; margin-right:auto; width:70%; border-radius:10px" }
+
+### Configure Cursor
+
+1. Open your Cursor desktop app.
+
+1. Navigate to **Settings** in the top right corner.
+
+1. In the **Cursor Settings** tab, go to **Tools & MCP** in the left sidebar.
+
+1. In the **Installed MCP Servers** section, click **New MCP Server**.
+
+1. Paste the following JSON configuration into the newly opened `mcp.json` tab:
+
+    ```json
+    {
+      "mcpServers": {
+          "weather": {
+              "url": "http://localhost:8000/mcp-listener"
+          }
+      }
+    }
+  ```
+
+1. Return to the **Cursor settings** tab. You should now see the `mcp-listener` MCP server with four tools available.
+
+1. To open a new Cursor chat, click <kbd>cmd</kbd> + <kbd>L</kbd> if you're on Mac, or <kbd>ctrl</kbd> + <kbd>L</kbd> if you're on Windows.
+
+1. In the Cursor chat tab, click **@ Add Context** and select `mcp.json`.
+
+
+### Validate aggregated MCP tools configuration
+
+You can now test each exposed tool using Cursor.
+
+{% navtabs "validate-mcp-tools" %}
```

Comment:
All the codeblocks in the navtabs are missing one space of indentation, this is affecting the numbering